### PR TITLE
Configure GitHub workflow for tests

### LINKED ISSUE #12: Add GitHub workflow to run basic dash smoke tests

### PR DESCRIPTION
**Pull request by alyssadai**
Ensures that basic linting + smoke tests of dash app are run by GitHub actions. 

Closes #12

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```
Comment (surchs) [inline .github/workflows/test.yml:25]: is this boilerplate? we have a `requirements.txt` here, do we need this if statement then?

File: tests/conftest.py [hunk 2]
```diff
@@ -1,6 +1,18 @@
 from pathlib import Path
 
 import pytest
+from selenium.webdriver.chrome.options import Options
```
Comment (surchs) [inline tests/conftest.py:4]: neat!

**Review by surchs — APPROVED** [review]
Looks good @alyssadai! I left one :cherries: question, but otherwise good to go :cook:

File: .github/workflows/test.yml (hunk 1 above)
Comment (alyssadai) [reply .github/workflows/test.yml:25]: yeah, it is - I had the same thought. Will remove for simplicity!

**Pull request merged**